optaplanner-persistence/optaplanner-persistence-jaxb/src/main/java/org/optaplanner/persistence/jaxb/api/score/buildin/PolymorphicScoreJaxbAdapter.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jaxb.api.score.buildin;
```

Comment:
The jackson version of this is in package  org.optaplanner.persistence.jackson.api.score.
So not in "buildin".
That should be consistent. Let's move this one, one level up. Or the other one to the buildin package (= BC breaking... so rather not)